packages/common/src/search/types/IHubSearchOptions.ts

```diff
@@ -1,5 +1,5 @@
 import { UserSession } from "@esri/arcgis-rest-auth";
-import { IArcGISContext, IHubRequestOptions, IModel } from "../..";
+import { EntityType, IArcGISContext, IHubRequestOptions, IModel } from "../..";
```

Comment:
imports that end in `.` result in circular dependencies

Reply:
fixed